# Thruster Mapper Parameters - OPTIMIZED FOR MOVEMENT

simple_thruster_mapper:
  ros__parameters:
    # Thrust limits
    max_thrust: 10.0
    min_thrust: -10.0
    
    # Scaling factor - INCREASED for stronger thrust
    thrust_scale: 5.0  # Was 3.0, now 5.0
    
    # Vertical thrust boost - NEW PARAMETER
    vertical_thrust_boost: 2.0  # Extra multiplier for Z-axis
    
    # Deadband (ignore small commands)
    deadband_threshold: 0.05
    
    # Rate limiting (prevent sudden changes)
    max_thrust_rate: 10.0  # Increased from 5.0
    
    # Individual thruster trim (for hardware calibration)
    thruster_trim:
      t1: 1.0  # Front-left
      t2: 1.0  # Front-right
      t3: 1.0  # Back-left
      t4: 1.0  # Back-right
      t5: 1.0  # Vertical-left
      t6: 1.0  # Vertical-right